CHANGES.rst

```diff
@@ -20,6 +20,8 @@ Bugfixes
 Accessibility
 ^^^^^^^^^^^^^
 
+- Reimplement Checkbox to make it programmatically focusable (:pr:`6528`, thanks :user:`foxbunny`)
+
 - Nothing so far :(
```

Comment:
that one is meant to be removed when adding the first real changelog line :p

Reply:
Sorry, didn't notice it.